src/components/PipelineRun/PipelineRunListView/PipelineRunListRow.tsx

```diff
@@ -46,6 +57,11 @@ const BasePipelineRunListRow: React.FC<React.PropsWithChildren<BasePipelineRunLi
     obj.metadata.labels = {};
   }
   const applicationName = obj.metadata?.labels[PipelineRunLabel.APPLICATION];
+  const gitProvider = obj.metadata?.annotations[PipelineRunLabel.COMMIT_PROVIDER_LABEL];
+  const repoOrg = obj.metadata?.labels[PipelineRunLabel.COMMIT_REPO_ORG_LABEL];
+  const repoURL = obj.metadata?.labels[PipelineRunLabel.COMMIT_REPO_URL_LABEL];
+  const prNumber = obj.metadata?.labels[PipelineRunLabel.PULL_REQUEST_NUMBER_LABEL];
+  const eventType = obj.metadata?.labels[PipelineRunLabel.COMMIT_EVENT_TYPE_LABEL];
```

Comment:
metadata will always have a value of obj is not undefined. you need optional chaining for labels and annotations
```
obj.metadata.labels?.[PipelineRunLabel.COMMIT_EVENT_TYPE_LABEL];
```